aframe/src/index.js

```diff
@@ -8,8 +8,9 @@ AFRAME.registerComponent('zesty-ad', {
   schema: {
     adSpace: { type: 'string' },
     creator: { type: 'string' },
-    adURI: { type: 'string' },
-    url: { type: 'string' },
+    defaultAd: { type: 'string', default: 'square' },
+    width: { type: 'float', default: 1 },
```

Comment:
This can be implicit from the height and the format, don't need to have it as a parameter.